controller/array_action/array_mediator_xiv.py

```diff
@@ -155,13 +159,37 @@ def get_volume(self, volume_name, pool_id=None):
 
     def get_volume_name(self, volume_id):
         try:
-            return self._get_vol_by_wwn(volume_id)
+            return self._get_object_name_by_wwn(volume_id)
         except xcli_errors.IllegalNameForObjectError as ex:
             logger.exception(ex)
             raise controller_errors.IllegalObjectName(ex.status)
 
+    def _expand_cli_volume(self, cli_volume, increase_in_blocks):
+        try:
+            self.client.cmd.vol_resize(vol=cli_volume.name, size_blocks=increase_in_blocks)
```

Comment:
AFAIK, vol_resize gets the new size and not the delta

Reply:
ok, it seems right. I fixed it

Reply:
the var name is still `increase_in_blocks`...